appcues/src/main/java/com/appcues/data/mapper/MapperExt.kt

```diff
@@ -2,14 +2,17 @@ package com.appcues.data.mapper
 
 import com.appcues.data.remote.response.action.ActionResponse
 import com.appcues.data.remote.response.trait.TraitResponse
+import com.appcues.trait.ExperienceTrait.ExperienceTraitLevel
 import java.util.UUID
 
-internal fun List<TraitResponse>.mergeTraits(other: List<TraitResponse>): List<TraitResponse> {
-    return mutableListOf<TraitResponse>().apply {
+internal typealias LeveledTraitResponse = Pair<TraitResponse, ExperienceTraitLevel>
```

Comment:
this was created soh at  merging time (TraitResponse) I already know its level, so that we can use it later when its all merged.

this was not necessary when @iujames did becasue of one change. the mergeTraits was merging ExperienceTraits instead of TraitResponse, and that was working because we had  each ExperienceTrait define a variable called type. (which I removed again)

Reply:
seems fine - typically would lean toward just having a simple data class to hold the items (plus allow for future extensibility) over using Pair - but also no issue with using Pair until if or when we actually need more than two things, maybe never.

Reply:
thats what came to mind to tie TraitResponse to a Level.